pkg/hubble/observer/option.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package serveroption
```

Comment:
Pinging @glibsm since I believe he had a rationale when he introduced this as a separate package.

Reply:
`serveroption` was moved into a separate package so other packages providing options don't pull in the entire dependency tree of the server

Reply:
I moved all options (observer, server, proxy) to option packages.